card/mod/03_machines/spec/set/type/javascript_spec.rb

```diff
@@ -6,23 +6,29 @@
   let(:changed_js)            { 'alert( "Hello" );' }
   let(:compressed_changed_js) { 'alert("Hello");'   }
 
-
   it_should_behave_like 'content machine', that_produces_js do
-    let(:machine_card)  { Card.gimme! "test javascript", type: :java_script, content: js}
+    let(:machine_card) do
+      Card.gimme! "test javascript", type: :java_script, content: js
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.